### PR TITLE
Create diameter.yml

### PR DESCRIPTION
**Pull request by Mesverrum**

### CODE REVIEW

**Comment by Mesverrum** [comment]
So since this is such an uncommon case and does not have a useful way to filter it by sysOID or sysDescr I've set this profile up to where it would need to be manually set in the snmp-base.yaml devices section.

File: profiles/kentik_snmp/broadforward/diameter.yml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      OID: 1.3.6.1.4.1.39216.1.2.20
+      name: dmtrPeerTable
+    symbols:
+      - OID: 1.3.6.1.4.1.39216.1.50.1.2.1.1.2
+        name: dmtrPeerAdminStatus
+        enum:
+          - inactive: 0
+          - active: 1
+      - OID: 1.3.6.1.4.1.39216.1.50.1.2.1.3
+        name: dmtrPeerDiameterId
```
Comment (thezackm) [inline profiles/kentik_snmp/broadforward/diameter.yml:29]: this is a string-value, we should move it to `metric_tags`

File: profiles/kentik_snmp/broadforward/diameter.yml [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+          closed: 0
+          established: 1
+      - OID: 1.3.6.1.4.1.39216.1.50.1.2.1.21
+        name: dmtrPeerRemoteId
+      - OID: 1.3.6.1.4.1.39216.1.50.1.2.1.22
+        name: dmtrPeerRealm
+      - OID: 1.3.6.1.4.1.39216.1.50.1.2.1.23
+        name: dmtrPeerLocalOriginId
+      - OID: 1.3.6.1.4.1.39216.1.50.1.2.1.24
+        name: dmtrPeerLocalOriginRealm
```
Comment (thezackm) [inline profiles/kentik_snmp/broadforward/diameter.yml:59]: these are all string-values, we should move them to `metric_tags`

File: profiles/kentik_snmp/broadforward/diameter.yml [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1141
+        name: dmtrPeerSctpOutOfSeqTSN
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1142
+        name: dmtrPeerSctpRxDups
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.1143
+        name: dmtrPeerSctpRxGapAcks
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.
+        name: dmtrPeerSctpMaxRTO
+      - OID: 1.3.6.1.4.1.39216.1.2.20.1.
+        name: dmtrPeerSctpMaxRTO
```
Comment (thezackm) [inline profiles/kentik_snmp/broadforward/diameter.yml:168]: the root OID on these seems to be wrong; should it be `1.3.6.1.4.1.39216.1.50.1.2.1.*` ? 

also; a few spot-checked as string values that need to be moved to `metric_tags`

**Review by thezackm — CHANGES_REQUESTED** [review]

File: profiles/kentik_snmp/broadforward/diameter.yml (hunk 1 above)
Comment (Mesverrum) [reply profiles/kentik_snmp/broadforward/diameter.yml]: right

File: profiles/kentik_snmp/broadforward/diameter.yml (hunk 3 above)
Comment (Mesverrum) [reply profiles/kentik_snmp/broadforward/diameter.yml]: fixed

File: profiles/kentik_snmp/broadforward/diameter.yml [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  - MIB: MF-DIAMETER-MIB
+    table:
+      OID: 1.3.6.1.4.1.39216.1.50.1.2.1
+      name: dmtrPeerTable
+    symbols:
+      - OID: 1.3.6.1.4.1.39216.1.50.1.2.1.1.2
+        name: dmtrPeerAdminStatus
+        enum:
+          - inactive: 0
+          - active: 1
```
Comment (thezackm) [inline profiles/kentik_snmp/broadforward/diameter.yml:27]: syntax needs the `- ` removed

**Review by thezackm — CHANGES_REQUESTED** [review]

**Review by thezackm — APPROVED** [review]

**Pull request merged**